nodeJS/orms/prisma_orm.md

```diff
@@ -118,6 +118,14 @@ You might have a complex query that you just are unable to get right via the Pri
 
 Prisma migrate is a tool that helps you perform database migrations. You won't be using it a whole ton in the curriculum, but it's good to be aware of it. When you decide to change the schema in any way, you run a Prisma migration to apply the schema changes to the database. These changes are tracked in a `migrations` folder in your codebase.
 
+<div class="lesson-note" markdown="1">
```

Comment:
A warning box would probably be more appropriate. I should probably bring up a style change to differentiate a standard note box and warning note box more than just the icon...
```suggestion
<div class="lesson-note lesson-note--warning" markdown="1">
```